x-pack/plugins/siem/server/lib/detection_engine/signals/filter_events_with_list.test.ts

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import uuid from 'uuid';
+import { filterEventsAgainstList } from './filter_events_with_list';
+import { mockLogger, repeatedSearchResultsWithSortId } from './__mocks__/es_results';
+
+import { ListClient } from '../../../../../lists/server';
+import { ListItemArraySchema } from '../../../../../lists/common/schemas';
+
+describe('filterEventsAgainstList', () => {
+  const someGuids = Array.from({ length: 13 }).map((x) => uuid.v4());
+
+  it('should respond with eventSearchResult if exceptionList is empty', async () => {
+    const res = await filterEventsAgainstList({
+      logger: mockLogger,
+      listClient: ({
+        getListItemByValues: async () => ([] as unknown) as ListItemArraySchema,
```

Comment:
You don't need the extra casting you can do:

```ts
getListItemByValues: async () => [],
```

And it will work

Reply:
I'm wondering if it's worth putting a listClientMock in the lists plugin? similar to what @rylnd suggested https://github.com/elastic/kibana/pull/65372#discussion_r423894227

Reply:
Yes, that is a great idea. Doesn't have to be on this PR unless you want to do it. Otherwise I will do it after this PR within lists.

Reply:
Okay I'll write up an issue and throw it in the backlog.